CONTRIBUTING.md

```diff
@@ -100,8 +100,8 @@ In order to improve the understandability of commands, we document all CLI optio
 
 
 ```diff
-- ansible-playbook -i production.yml app.yml -l france -t stop
-+ ansible-playbook --inventory production.yml app.yml --limit france --tag stop
+- npm start -s [$service_id] -r
++ npm start --services [$service_id] --refilter-only
```

Comment:
Same as above, this would not work and is more confusing for the pursued goal:

```suggestion
- ota track -s [$service_id] -r
+ ota track --services [$service_id] --refilter-only
```